training/model_management/src/azureml/model/mgmt/processors/transformers/whisper/predict.py

```diff
@@ -196,6 +199,14 @@ def predict(
     """
     if not task == "automatic-speech-recognition":
         return f"Invalid task name {task}"
+    
+    device = kwargs.get("device", -1)
```

Comment:
Why not always set `device=cuda` if  `torch.cuda.is_available()`?

Reply:
Right now, all the other models in the registry take the device as parameter from the user, so we are maintaining the same behaviour for whisper as well. Also to keep the scenario of a user wanting to use GPU sku with CPU device open.